crates/rust-analyzer/src/semantic_tokens.rs

```diff
@@ -70,7 +70,7 @@ macro_rules! define_semantic_token_modifiers {
 }
 
 define_semantic_token_modifiers![
-    (CONSTANT, "constant"),
+    (CONSTANT, "readonly"),
```

Comment:
Hm, I guess this is "practicality beats purity"? const & readonly are pretty different things, but I guess we can re-purpose `readonly` as `const` and just use `mutable` for (complement of) cases where we'd use `readonly` otherwise?

Reply:
I think this is debatable. It's not like we can match the builtin scopes 100%, and `constant` is more semantically correct, so we could just decide to do our own thing. It does seem like the builtins are basically "whatever typescript needs."

Reply:
I am inclined to stick to semantically-correct constant, unless readonly significantly improves theme support.